Document that some august entities are slower to update (#28645)
home-assistant/home-assistant.github.io: source/_integrations/august.markdown

		
Other August locks expect to be powered by AA alkaline (non-rechargeable) batteries. Rechargeable batteries in these locks will result in incorrect reporting of battery charge.

## Push updates not available for some entities

> While most entities are able to be updated via the push API, August/Yale does not offer a push API for for some data which means these entities will update slower:

Author: ```suggestion
While most entities can be updated via the push API, August/Yale does not offer a push API for some data, which means these entities will update slower:
```

> - Doorbell ding sensor (Doorman models only)

Author: While this is true for current as well, the doorman support is new and since this isn't critical information (mainly just to reduce issues), I targeted next
